deployment/data-streams/changeset/deploy.go

```diff
@@ -0,0 +1,62 @@
+package changeset
+
+import (
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+)
+
+type (
+	// Contract covers contracts such as channel_config_store.ChannelConfigStore and fee_manager.FeeManager.
+	Contract interface {
+		// Caller:
+		Owner(opts *bind.CallOpts) (common.Address, error)
+		SupportsInterface(opts *bind.CallOpts, interfaceID [4]byte) (bool, error)
+		TypeAndVersion(opts *bind.CallOpts) (string, error)
+
+		// Transactor:
+		AcceptOwnership(opts *bind.TransactOpts) (*types.Transaction, error)
+		TransferOwnership(opts *bind.TransactOpts, to common.Address) (*types.Transaction, error)
+	}
+
+	ContractDeployFn[C Contract] func(chain deployment.Chain) *ContractDeployment[C]
+
+	ContractDeployment[C Contract] struct {
+		Address  common.Address
+		Contract C
+		Tx       *types.Transaction
+		Tv       deployment.TypeAndVersion
+		Err      error
+	}
+)
+
+var _ deployment.ChangeSetV2[DeployChannelConfigStoreConfig] = DeployChannelConfigStore{}
+
+// deployContract deploys a contract and saves the address to the address book.
+//
+// Note that this function modifies the given address book variable, so it should be passed by reference.
+func deployContract[C Contract](
```

Comment:
Why copy this one over from helpers vs. just calling it from there? https://github.com/smartcontractkit/chainlink/blob/b85e6c8c10e09c53394db58d00b0cd6504113d8a/deployment/helpers.go#L157-L157